wcfsetup/install/files/lib/data/user/online/UserOnline.class.php

```diff
@@ -215,9 +215,14 @@ public function getBrowser() {
 		}
 		
 		// edge
-		if (preg_match('~edge/(\d{2}\.\d+)~i', $this->userAgent, $match)) {
+		if (preg_match('~edge/(\d{2}\.\d+)|edg/([\d\.]+)~i', $this->userAgent, $match)) {
 			return 'Microsoft Edge '.$match[1];
```

Comment:
I believe this will not return a proper version number for the Chromium based edge, as the version will be captured by a distinct capture group.

Reply:
Yeah, there stands just "Microsoft Edge" without any version